### PR TITLE
[Tizen] Fix slider thumb color issue

### PR DESCRIPTION
**Pull request by shyunMin**
### Description of Change ###
This PR is to fix the slider thumb button issue.

### Issues Resolved ### 

- fixes #

### API Changes ###

 
 None

### Platforms Affected ### 

- Tizen

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

- Before
s1

- After
s2

### PR Checklist ###

- [ ] Has automated tests 
- [x] Rebased on top of the target branch at time of PR
- [x] Changes adhere to coding standard

### CODE REVIEW

File: Xamarin.Forms.Material.Tizen/MaterialSliderRenderer.cs [hunk 1]
```diff
@@ -17,5 +17,12 @@ protected override void OnElementChanged(ElementChangedEventArgs<Slider> e)
 			}
 			base.OnElementChanged(e);
 		}
+
+		protected override void UpdateThumbColor()
+		{
+			var color = Element.ThumbColor.IsDefault ? MColors.Current.PrimaryColor : Element.ThumbColor.ToNative();
+			Control.SetPartColor("handler", color);
```
Comment (rookiejava) [inline Xamarin.Forms.Material.Tizen/MaterialSliderRenderer.cs]:
```suggestion
Control.SetPartColor(Parts.Slider.Handler, color);
```

File: Xamarin.Forms.Material.Tizen/MaterialSliderRenderer.cs [hunk 2]
```diff
@@ -17,5 +17,12 @@ protected override void OnElementChanged(ElementChangedEventArgs<Slider> e)
...
+			Control.SetPartColor("handler_pressed", color);
```
Comment (rookiejava) [inline Xamarin.Forms.Material.Tizen/MaterialSliderRenderer.cs]:
```suggestion
Control.SetPartColor(Parts.Slider.HandlerPressed, color);
```

File: Xamarin.Forms.Material.Tizen/MaterialSliderRenderer.cs (hunk 1 above)
Comment (shyunMin) [reply Xamarin.Forms.Material.Tizen/MaterialSliderRenderer.cs]: Fixed.

File: Xamarin.Forms.Material.Tizen/MaterialSliderRenderer.cs (hunk 2 above)
Comment (shyunMin) [reply Xamarin.Forms.Material.Tizen/MaterialSliderRenderer.cs]: Fixed.

**Pull request merged**